arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -958,6 +958,9 @@
             "$ref": "#/parameters/LocationParameter"
           },
           {
+            "$ref": "#/parameters/TagsParameter"
```

Comment:
All the parameters that you are trying to add are already present as properties of [VirtualMachine](https://github.com/Azure/azure-rest-api-specs/pull/558/commits/a15212d63d1dbacdc500c284712e3271073cf63d#diff-32c44936fe36d535f8c42dcb1ab29264R953)